src/indieweb_utils/webmentions/send.py

```diff
@@ -62,9 +67,17 @@ def send_webmention(source, target, me=None):
 
     valid_status_codes = (200, 201, 202)
 
-    if r.status_code in valid_status_codes:
-        message = {"title": message, "description": message, "url": target, "status": "success"}
-    else:
-        message = {"title": "Error: " + message, "description": "Error: " + message, "url": target, "status": "failed"}
-
-    return message
+    if r.status_code not in valid_status_codes:
+        return SendWebmentionResponse(
+            title=f"Error: {message}",
+            description=message,
+            url=target,
+            succeeded=False,
```

Comment:
Other attributes (title, description, url) of SendWebmentionResponse are nouns, so we should probably rename succeeded -> success to keep it consistent.
```suggestion
            success=False,
```